networking_cisco/tests/unit/cisco/etc/cfg_syncer/neutron_router_db_aci.json

```diff
@@ -0,0 +1 @@
oups": [], "binding:vif_details": {"ovs_hybrid_plug": true, "port_filter": true}, "binding:vif_type": "ovs", "mac_address": "fa:16:3e:97:ec:f2", "status": "ACTIVE", "hosting_info": {"hosting_mac": "fa:16:3e:97:ec:f2", "hosting_port_id": "527b7bc0-d6ba-4d07-932a-60e96da86173", "physical_interface": "Port-channel10", "segmentation_id": 3000, "hosting_port_name": ""}, "binding:host_id": "asr1k-k7-controller-1-R2", "ha_info": {"group": "1064", "ha_port": {"status": "DOWN", "subnets": [{"ipv6_ra_mode": null, "cidr": "172.16.0.0/16", "gateway_ip": "172.16.0.1", "id": "8f9c885d-a2ad-4fff-8127-2d4b3af6015b"}], "binding:host_id": "", "name": "", "allowed_address_pairs": [], "admin_state_up": true, "network_id": "7a38a5fd-3ad9-4952-ab08-4def22407269", "fixed_ips": [{"prefixlen": 16, "subnet_id": "8f9c885d-a2ad-4fff-8127-2d4b3af6015b", "ip_address": "172.16.0.104"}], "binding:vif_details": {}, "binding:vnic_type": "normal", "binding:vif_type": "unbound", "device_owner": "network:router_gateway", "tenant_id": "", "mac_address": "fa:16:3e:77:ae:bb", "id": "b768491c-4b76-492f-a0ee-56bc50867a07", "binding:profile": {}, "extra_subnets": [], "extra_dhcp_opts": [], "security_groups": [], "device_id": "a2617b46-78ef-49ba-95b5-0f3ff90f1f91"}, "other_config": "", "timers_config": "", "tracking_config": "", "type": "HSRP"}, "device_id": "213fd355-78c3-44cc-bd6e-492b9ade2602", "name": "", "admin_state_up": true, "network_id": "7a38a5fd-3ad9-4952-ab08-4def22407269", "tenant_id": "", "binding:vnic_type": "normal", "extra_subnets": []}, "routertype-aware-scheduler:share_hosting_device": true, "cisco_ha:details": {"redundancy_routers": [{"priority": 97, "state": "STANDBY", "id": "213fd355-78c3-44cc-bd6e-492b9ade2602"}, {"priority": 100, "state": "STANDBY", "id": "15315cc4-d00d-446d-bf6e-b077faaefaa1"}], "probe_connectivity": false, "priority": 100, "state": "ACTIVE", "redundancy_level": 2, "type": "HSRP"}, "id": "213fd355-78c3-44cc-bd6e-492b9ade2602", "share_host": true, "status": "ACTIVE", "routerhost:hosting_device": "00000000-0000-0000-0000-000000000003", "routes": [], "router_type": {"cfg_agent_service_helper": "neutron.plugins.cisco.cfg_agent.service_helpers.routing_svc_helper.RoutingServiceHelper", "cfg_agent_driver": "neutron.plugins.cisco.cfg_agent.device_drivers.asr1k.aci_asr1k_routing_driver.AciASR1kRoutingDriver", "name": "ASR1k_router", "id": "00000000-0000-0000-0000-000000000003"}, "routertype-aware-scheduler:auto_schedule": false, "cisco_ha:enabled": true, "hosting_device": {"status": "ACTIVE", "booting_time": 360, "host_category": "Hardware", "credentials": {"password": "cisco123", "user_name": "admin"}, "protocol_port": 22, "id": "00000000-0000-0000-0000-000000000003", "management_ip_address": "172.20.231.19", "service_types": "router:FW:VPN", "name": "ASR1k template", "admin_state_up": true, "created_at": "2015-11-16 22:54:18", "timeout": null, "template_id": "00000000-0000-0000-0000-000000000003"}, "routertype:id": "00000000-0000-0000-0000-000000000003"}]
```

Comment:
I will investigate whether we can use the existing DB file -- the only difference between this DB file and the existing one is that this passes the correct configuration agent driver (AciASR1k, instead of ASR1k). However, I don't think this ever gets used, and if that's the case, we'll just use the existing DB file.